twig/utils/filesystem.py

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+
+SOURCES_GLOB = "**/*.vy"
+
+
+def collect_sources(path, glob=SOURCES_GLOB):
```

Comment:
In theory, this function would be a public API which would imply it should live outside of `utils` (which should be `_utils` under our new paradigm)